include/z64save.h

```diff
@@ -462,8 +462,23 @@ typedef enum LinkAge {
 
 /*
  * SaveContext.eventChkInf
+ * eventChkInf is an array of 16 bit flags. These typically track main quest events.
+ * Generally, a flag will be assigned a unique id, ranging from 0x00-0xDE. Flag state can be individually accessed
+ * or modified by passing the flag id into GET_EVENTCHKINF, SET_EVENTCHKINF, and CLEAR_EVENTCHKINF
+ *
+ * In some instances where a set of flags share a common accociation, the eventChkInf variable will be accessed directly.
```

Comment:
should `GET_EVENTCHKINF_VAR` not be called `VAR` then?

Reply:
I think these can be called "page". As in each page has 16 flags in it.
But if that isnt popular, yeah I like "entry" over "variable".

Reply:
tbh though even entry can make it sound like you mean a single flag. page is pretty nice at conveying that its a grouping of many things, like many words on a page (or flags in a page in this case)

Reply:
still like page more personally, element still sounds like a single thing while page conveys its a single grouping

Reply:
in that case even suffix `_INDEX` can be exchanged for `_PAGE` too (again, maybe not for this PR)

thinking about it, `GET_EVENTCHKINF_VAR` doesn't return an individual unit of data; a real variable might be suballocated in that "page". for instance, with another array
```
enum IngoRaceState
EVENTINF_INGORACE_INDEX
EVENTINF_INGORACE_STATE_MASK
```
is a `4 bits` in `page 0` of `gSaveContext.eventInf`

Reply:
I really dislike using page here because it doesn't fit well to the existing definitions of a page. If we're talking data sizes, I wouldn't expect a page to be smaller than a single word.

I guess I also dislike entry/element in that the big picture idea is that `eventChkInf`, `itemGetInf`, `infTable`, `eventInf` should be bit arrays and that we should prioritize accessing them as such; in a bit array, one bit is one element in the array. 

i looked to see if there was some special name for the type used to hold the bit array, but the only implementation I found just called it "CHAR", the type used to contain the bits.

Reply:
I dont see why page has to be a word in length or why size has to do with anything at all. its just a higher level name for the grouping of each sets of flags, where each index is a page number into a "book" of flags

Reply:
One of the definitions for page relates to it's size as a unit of memory. Merriam-Webster lists one definition of page as "a sizable subdivision of computer memory". In 6502 assembly for example, one page is a 256 byte aligned block of memory, with the "zero page" memory being located at memory $0000 to $00FF inclusive. A halfword of data isn't exactly a "sizable subdivision of memory".

More than that though, I've never heard of an array being referred to a book of pages (or a group of flags being a page for that matter), and I don't think the analogy makes much sense. One of the limitations of a physical book is that there is a limit to the number of pages that can be viewed at a given time. The memory that an event flag is located at is always freely accessible (to the graph thread at least).

Reply:
I think its pretty clear that the context is not talking about page as a unit or subdivision of memory.

You dont even have to think about it as a book-- if you were to visualize these flags with a menu for example (like gz does) you can certainly see them as different pages that you have to click/scroll through. It just gives a physical name we can reference these as, instead of coding constructs like "elements of an array" or "variables".

Reply:
I don't see anything wrong with referring to coding constructs in coding construct terms, and I'd prefer it that way because ideally we should AVOID exposing the underlying variables whenever possible, and make the GET/SET/CLEAR flag macros the main way you retrieve data from the bit arrays. It's just that with decomp we are just forced expose more than this because we can't change codegen.

Your argument also doesn't convince me. Yes, if you're creating a UI it makes sense to think of your different views as being different pages in a document. But we're talking about naming the underlying data itself, and I don't find it to be very "page-like" for the reasons I've already given. Like in gz or `flg_set.c` you might have an editor that flips through the variables individually for editing, but you could just as well display more than one variable at a time. Then what? Are they still pages? Do you describe the view as being a page of pages?